### PR TITLE
Docker enhancement

### PR DESCRIPTION
**Pull request by DanielGuramulta**

### CODE REVIEW

File: ci/travis/inside_docker.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PROTOBUF_INSTALL_DIR="/aditof-deps/installed/protobuf"
+WEBSOCKETS_INSTALL_DIR="/aditof-deps/installed/websockets"
+
+mkdir -p build
+pushd build
+cmake .. ${cmake_option} -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="${GLOG_INSTALL_DIR};${PROTOBUF_INSTALL_DIR};${WEBSOCKETS_INSTALL_DIR}"
+make -j${NUM_JOBS}
+popd #build
+
+popd # ${PROJECT}
```
Comment (dNechita) [inline ci/travis/inside_docker.sh]: PROJECT variable no longer exists

File: ci/travis/build_docker_image.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+mkdir -p temp_deps
+
+get_deps_source_code temp_deps
+
+dockername=$1
+dockerfilepath=$2
+
+docker build -t ${dockername} -f ${dockerfilepath} .
+
+rm temp_deps
```
Comment (dNechita) [inline ci/travis/build_docker_image.sh]: rm -r

File: ci/travis/inside_docker.sh (hunk 1 above)
Comment (DanielGuramulta) [reply ci/travis/inside_docker.sh]: The variable was renamed to project_dir, modified the comment.

File: ci/travis/build_docker_image.sh (hunk 2 above)
Comment (DanielGuramulta) [reply ci/travis/build_docker_image.sh]: Fixed now. Thank you!

**Pull request merged**